cmd/operator-sdk/cli/cli.go

```diff
@@ -33,9 +34,9 @@ import (
 )
 
 var commands = []*cobra.Command{
-	// Once the KB CLI is made the default, add the "new" command as a way to
-	// scaffold the legacy project layout and mark "new" as deprecated
-	// new.NewCmd()
+	// The "new" cmd provides a way to scaffold Helm/Ansible projects
+	// from the new CLI.
+	new.NewCmd(),
 
 	alpha.NewCmd(),
 	build.NewCmd(),
```

Comment:
The `operator-sdk create api` is not showing when we use the help. It might be solved in a follow-up. wdyt @joelanford @estroz ? Should we not show that in the help?

Reply:
That's a plugin issue right @estroz 
You have to be inside an initialized project for that help to show up for the correct plugin version I believe.
```console
$ operator-sdk create api -h
Scaffold a Kubernetes API.

For project-specific information, run this command in the root directory of a
project.

Note: unable to find configuration file, project must be initialized

Usage:
  operator-sdk create api [flags]

Flags:
  -h, --help   help for api
```

And inside a project:
```console
$ operator-sdk create api -h
Scaffold a Kubernetes API by creating a Resource definition and / or a Controller.

create resource will prompt the user for if it should scaffold the Resource and / or Controller.  To only
scaffold a Controller for an existing Resource, select "n" for Resource.  To only define
the schema for a Resource without writing a Controller, select "n" for Controller.

After the scaffold is written, api will run make on the project.

Usage:
  operator-sdk create api [flags]

Examples:
  # Create a frigates API with Group: ship, Version: v1beta1 and Kind: Frigate
  operator-sdk create api --group ship --version v1beta1 --kind Frigate

...
```

Reply:
You need to set `--project-version` to output help for the default plugin. However I do see `create` listed in the available subcommands when I run `operator-sdk -h` regardless of where the binary is.

Reply:
create      Scaffold a Kubernetes API or webhook 👍